plugins/video/pkg_src/CMakeLists.txt

```diff
@@ -70,12 +70,19 @@ find_library(cuda_LIBRARY cuda
              PATHS ${CMAKE_CUDA_IMPLICIT_LINK_DIRECTORIES}
              PATH_SUFFIXES lib/stubs lib64/stubs)
 
+if(NOT $ENV{LD_PRELOAD} STREQUAL "")
+    set(NEW_LD_PRELOAD "$ENV{LD_PRELOAD} ${cuda_LIBRARY}")
+else()
+    set(NEW_LD_PRELOAD "${cuda_LIBRARY}")
+endif()
+string(REPLACE " " ":" NEW_LD_PRELOAD "${NEW_LD_PRELOAD}")
```

Comment:
Why we can't just `set(NEW_LD_PRELOAD "$ENV{LD_PRELOAD}:${cuda_LIBRARY}")` to have the colon already there? Also, this will break if somebody has a space in their `LD_PRELOAD` or cuda path/

Reply:
the thing is that the LD_PRELOAD has a space, and my command below doesn't like it. So I am using an alternative way to separate paths.